tools/ci/ci_built_diff.sh

```diff
@@ -18,6 +18,7 @@ main() {
     )
 
     ./update-built-tests.sh
+    python ./mimesniff/mime-types/resources/generated-mime-types.py
```

Comment:
Oh, sorry this should go in `update_built_tests.sh`. I guess the distinction is to make it easy to bulk update all the tests.

Reply:
Changed!